### ISSUE TITLE
Regression in 0.7.11 for certain devices

### ISSUE THREAD

**Issue by zusamann** [faisalman/ua-parser-js#199]
Sample User Agent:
```
Mozilla/5.0 (Linux; Android 6.0.1; A0001 Build/MHC19Q) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.85 Mobile Safari/537.36
```

Output from 0.7.10
```
{ ua: 'Mozilla/5.0 (Linux; Android 6.0.1; A0001 Build/MHC19Q) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.85 Mobile Safari/537.36',
  browser: { name: 'Chrome', version: '54.0.2840.85', major: '54' },
  engine: { version: '537.36', name: 'WebKit' },
  os: { name: 'Android', version: '6.0.1' },
  device: { model: undefined, vendor: undefined, type: 'mobile' },
```

Output from 0.7.11
```
{ ua: 'Mozilla/5.0 (Linux; Android 6.0.1; A0001 Build/MHC19Q) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.85 Mobile Safari/537.36',
  browser: { name: 'Chrome', version: '54.0.2840.85', major: '54' },
  engine: { version: '537.36', name: 'WebKit' },
  os: { name: 'Android', version: '6.0.1' },
  device: { vendor: undefined, model: undefined, type: undefined },
  cpu: { architecture: undefined } }
```

As you can see in this instance the device.type is returning mobile in 0.7.10 and undefined in 0.7.11

**Comment by faisalman** [comment]
This was fixed in #87

**Comment by zusamann** [comment]
When will the fix start showing up on `npm i ua-parser-js`? Right now I've locked the version number in my package.json so that on redeploys it works fine.

**Comment by faisalman** [comment]
This patch will be released in version `0.7.12`, possibly sometime in next week
